packages/volto/locales/volto.pot

```diff
@@ -4895,11 +4890,6 @@ msgstr ""
 msgid "target path"
 msgstr ""
 
-#. Default: "Text"
-#: config/Blocks
-msgid "text"
-msgstr ""
```

Comment:
Shouldn't this be kept for the slate block? It's currently using a hardcoded title instead of a message: https://github.com/plone/volto/blob/7995bfcc45701389ce2f2b267c0069cbd021390f/packages/volto-slate/src/blocks/Text/index.js#L107